wdl2cwl/main.py

```diff
@@ -217,8 +218,21 @@ def main(argv: List[str]) -> str:
     # ^ converts multine line strings to nice multiline YAML
     yaml.dump(cwl_result, result_stream)
     yaml.dump(cwl_result, sys.stdout)
+
     return result_stream.getvalue()
 
 
+def main() -> None:
+    """Command-line parsing."""
+    parser = argparse.ArgumentParser()
+    parser.add_argument("workflow", help="WDL workflow")
+    parser.add_argument("-d", "--directory", help="Directory to store CWL files")
+    args = parser.parse_args()
+
+    if args.directory:
+        with open(args.directory, "w") as result:
```

Comment:
This creates a file with the name from `args.directory`. Shouldn't it create a file **inside** that directory?